### PR TITLE
SEER homepage

### PR DESCRIPTION
**Pull request by BroYash**
No functionality just GUI

### CODE REVIEW

File: frontend/src/Components/Nav.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Homepage from './pages/Homepage';
+
+function Nav() {
```
Comment (kelly-luo) [inline frontend/src/Components/Nav.js:5]: Function name could be improved as it is quite ambiguous

Comment (BroYash) [reply frontend/src/Components/Nav.js:5]: navigation

Comment (kelly-luo) [reply frontend/src/Components/Nav.js:5]: How will this link to the homepage?

Comment (BroYash) [reply frontend/src/Components/Nav.js:5]: it will link to every page

**Review by kelly-luo — APPROVED** [review]
After cloning and running the code, there is no error and the page is successful

**Pull request merged**